tests/regression_tests/weightwindows/test.py

```diff
@@ -108,7 +109,7 @@ def model():
 
 
 def test_weightwindows(model):
-    test = HashedPyAPITestHarness('statepoint.2.h5', model)
+    test = TolerantPyAPITestHarness('statepoint.2.h5', model)
```

Comment:
Thanks for updating the results! Why the change to the tolerance test harness? The results should still be reproducible right?